GameEngineTests/Players/EpsilonGreedyPlayerTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using GameEngine;
+using GameEngine.Players;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GameEngineTests.Players
+{
+    [TestClass]
+    public class EpsilonGreedyPlayerTests
+    {
+        private CardType[] OneCardOfEachColor
+        {
+            get
+            {
+                return new Deck(1).Cards
```

Comment:
❌ I don't like this because it relies on assumptions about how the Deck is initialized. I prefer the version of this helper that uses the enumeration.

Reply:
When #3 is merged, I'll fix it here 😁

Reply:
Fixed!